ansible_base/rbac/policies.py

```diff
@@ -18,7 +18,10 @@ def visible_users(request_user, queryset=None) -> QuerySet:
     if has_super_permission(request_user, 'view') or (
         get_setting('ORG_ADMINS_CAN_SEE_ALL_USERS', False) and org_cls.access_ids_qs(request_user, 'change').exists()
     ):
-        return user_cls.objects.all()
+        if queryset is not None:
+            return queryset
+        else:
+            return user_cls.objects.all()
```

Comment:
We have an identical condition later on in this method, we might want to combine those into 1 conditional at the start that may redefine `queryset`. But this should be perfectly functional